Data/TypeDocumentation.cs

```diff
@@ -190,6 +190,11 @@ void XmlToHtml (XElement x, TextWriter w)
                         }
                     }
                     break;
+                case "typeparamref":
+                    w.Write ("<span class=\"inline-code c-tr\">");
+                    w.Write (x.Attribute ("name")?.Value ?? "");
```

Comment:
Please use `WriteEncodedHtml (x.Attribute ("name")?.Value ?? "", w);` to make sure the name gets escaped.

Reply:
Done.